dashboard/src/index.css

```diff
@@ -1,2 +1,3 @@
 @import './assets/Inter/inter.css';
 @import 'frappe-ui/src/style.css';
+@import url('https://fonts.googleapis.com/css2?family=Fira+Code:wght@300..700&display=swap');
```

Comment:
Hopefully this is not being used in UI elements.. is it?

Reply:
Let's stick to FrappeUI theme as is for dashboard

Reply:
Changed